content/participate/report-issue.adoc

```diff
@@ -0,0 +1,202 @@
+
+First, determine what kind of issue it is. There are three main types
+whose use is explained on this page:
+
+* *Bug*: Something does not work as intended.
+* *New Feature*: You want Jenkins to have new functionality.
+* *Improvement*: Existing functionality should be changed.
+
+Other types (_Patch_, _Task_) are not widely used in JIRA. Neither will
+be explained here. If you want to propose a patch, create an issue of
+one of the 3 types above and then create a link:https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/creating-a-pull-request[pull request in GitHub].
+
+[[Howtoreportanissue-Beforereportingabug]]
+=== Before reporting a bug
+
+* First, *search for the error message or general problem description*.
+This can help you find solutions if it's actually e.g. a configuration
+problem, or point you to earlier reports in case you're not the first
+one experiencing this problem. You can
+http://issues.jenkins-ci.org/secure/IssueNavigator.jspa[use the search
+in the issue tracker], or use web search.
+https://stackoverflow.com/tags/jenkins[Stack Overflow] may also be a
+good place to search. Any results will often point you to earlier issue
+reports if it's really a bug. You can then _watch them_ and _vote for
+them_ in JIRA. Voting helps us determine the impact of bugs and
+prioritize them. _If you have doubts that the problem you're
+experiencing is the exact same one as you found, prefer to file a new
+issue (and mention the one(s) you found in the description)._
+* *Make sure that the problem actually is with Jenkins*. If your build
+seems to fail for no reason, try to build your project outside Jenkins,
+but in the same general environment (e.g. same machine, same user
+account, ...). If Jenkins interacts with other systems, make sure that
+they're working. If the connections between Jenkins master and agents
+fail, make sure the network is reliable.
+* *Check the changelogs* to see whether the issue has been fixed in
+newer versions of the affected component.
+For Jenkins itself, the link:/changelog-stable/[long term support changelog], the link:/doc/upgrade-guide/[upgrade guide], and the link:/changelog/[weekly changelog] is http://jenkins-ci.org/changelog[here].
+If you believe the problem to be in a plugin, review the changelog for the specific plugin on the link:https://plugins.jenkins.io/[plugins site].  The plugin documentation can also be reached
+directly by clicking the plugin name in _Manage Jenkins_ » _Manage Plugins_ » _Installed_.
+If you're still unable to resolve the problem and cannot find existing
+issue reports or related fixes in newer versions of Jenkins or the
+plugins, file a *bug* in link:https://issues.jenkins-ci.org/[JIRA].
+
+[[Howtoreportanissue-Beforerequestinganimprovementorfeature]]
+=== Before requesting an improvement or feature
+
+* Check whether newer versions of Jenkins or the plugin in question have
+been released. They may have the feature already!
```

Comment:
Following the same pattern of bold text for the key phrase as is used in the next bullet item and in the preceding bullet items
```suggestion
* *Check newer versions of Jenkins or the plugin* for this feature.
They may have the feature already!
```